Description: Consume newline characters after prompt
 When prompting for user input, consume characters up to the
 newline character; avoids leaving characters behind that end
 up being consumed by the shell after gptsync terminates.
Author: Julien BLACHE <jblache@debian.org>

Index: refit-0.14/gptsync/os_unix.c
===================================================================
--- refit-0.14.orig/gptsync/os_unix.c	2010-04-19 18:30:26.094931554 +0200
+++ refit-0.14/gptsync/os_unix.c	2010-04-19 18:30:19.863932083 +0200
@@ -150,7 +150,11 @@
         printf("No\n");
         *bool_out = FALSE;
     }
-    
+
+    /* Consume everything up to and including newline */
+    while (getchar() != '\n')
+      ;
+
     return 0;
 }
 
